src/test/kotlin/org/rustSlowTests/ide/actions/RsBuildActionTest.kt

```diff
@@ -31,41 +32,51 @@ class RsBuildActionTest : CargoBuildTest() {
 
     fun `test build first action`() {
         val testProject = fileTree {
-            toml("Cargo.toml", """
+            toml(
```

Comment:
I think you don't want to format this code

Reply:
Fixed.